test/FunctionalTests/Microsoft.OData.Client.Tests/Microsoft.OData.Client.Tests.Netcore.csproj

```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="FluentAssertions" Version="4.19.2" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
     <PackageReference Include="Microsoft.NETCore.Portable.Compatibility" Version="1.0.1" />
+    <PackageReference Include="Moq" Version="4.10.1" />

```

Comment:
Moq version should be the latest `4.13.1` just like the one you have used in `Microsoft.OData.Client.Tests` project I believe. And again, do we have any test case added in this project that requires `Moq`?

Reply:
Yes. the lastest Moq version(4.13.1) supports netcoreapp2.0 but this project targets netcoreapp1.0 and this is the most up to date Moq version that is compatible with netcoreapp1.0

Reply:
Yes there is a sample test to shoe how these methods can be mocked.

Reply:
In that case is it good to consistently use version `4.10.1` in all projects? I can only see tests added for `Microsoft.OData.Client.Tests` project and not `Microsoft.OData.Client.Tests.Netcore` project. This brings the question if we need `Moq `in `Microsoft.OData.Client.Tests.Netcore` project.

Reply:
Basically for the test to run in a .netcore application, then the <code>Moq framework </code> 
is needed in <code>Microsoft.OData.Clieny.Tests.Netcore</code>

Reply:
I don't see why we cannot use the latest <code>Moq Framework </code> in <code>
Microsoft.OData.Client.Tests</code> if it is supported.